entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -123,7 +124,7 @@ class EntityOperationHandler(
             .zipWith(extractSubjectOrEmpty())
             .map { entitiesIdUserId ->
                 val (existingEntities, unknownEntities) = entityOperationService
-                    .splitEntitiesIdsByExistence(entitiesIdUserId.t1)
+                    .splitEntitiesIdsByExistence(entitiesIdUserId.t1.map { it.toUri() })
```

Comment:
a case for the sister function `List<String>.toUri()`?

Reply:
indeed